pulp_2to3_migration/app/pre_migration.py

```diff
@@ -625,12 +604,9 @@ def handle_outdated_resources(plan):
     for plugin_plan in plan.get_plugin_plans():
         inplan_repos = plugin_plan.get_repositories()
```

Comment:
Should we make this a set to begin with?  I think it's a list, but I'm not sure of any particular reason why it needs to be a list, and it might have duplicate entries for some IDs.

Reply:
Yeah, we can make it a tuple, technically it would be an invalid/wrong plan if it had duplicated repo ids listed.
This list is specifically for repositories and it doesn't include importer or distributor ones.

Reply:
Not targeted specifically at the repository list - actually it's the importer and distributor lists where it's more obvious how duplicates might sneak in.